nav_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
```

Comment:
A path here is not portable. I would suggest that this be a set map method not a load map method. If you need to load a map from file, there can be a standalone program to load it and send it.